pandas/sparse/frame.py

```diff
@@ -187,9 +213,47 @@ def _init_matrix(self, data, index, columns, dtype=None):
         if len(index) != N:
             raise ValueError('Index length mismatch: %d vs. %d' %
                              (len(index), N))
+        return index, columns
 
-        data = dict([(idx, data[:, i]) for i, idx in enumerate(columns)])
-        return self._init_dict(data, index, columns, dtype)
+    def to_coo(self):
+        """
+        Convert the frame to its SciPy sparse COO matrix representation.
```

Comment:
does this require any special index? mention either way (and what happens to the index anyhow)? what about column names?